### ISSUE TITLE
Add .english.md to english rosetta challenges

### ISSUE THREAD

**Issue by moT01** [freeCodeCamp/freeCodeCamp#38757]
A number of the Rosetta code challenge files end with `.md`. They should end with `.english.md`.

**Comment by UgRoss** [comment]
Hey @moT01,
can I take this?
Will open PR tomorrow

**Comment by UgRoss** [comment]
Files list
<img width="868" alt="CleanShot 2020-05-07 at 22 10 54@2x" src="https://user-images.githubusercontent.com/5703885/81335146-ceb70980-90af-11ea-8556-9315dda0ce99.png">

**Comment by moT01** [comment]
That looks correct @UgRoss, you may want to wait on the PR until https://github.com/freeCodeCamp/freeCodeCamp/pull/38747 gets merged since it's all the same files.

**Comment by UgRoss** [comment]
Sure, thanks!

**Comment by chaos3737** [comment]
Hey there @moT01!
I'm super new to coding and looking for open projects and teams to join. Any tips or help is greatly appreciated!

**Comment by rajat641** [comment]
Hi @moT01 , I can work on this.I have coding experience in Python and Java but have no experience in open source projects. Please let me know if I can work on this.

**Comment by moT01** [comment]
I mentioned that there's another PR in place to add something to most of these files, and that it might be a good idea to wait for that one to get merged. Anyone is free to make a PR, but you will likely have to rebase and make the same changes again after that one gets merged.

**Comment by rajat641** [comment]
@moT01 Thanks for that update. I have got another issue to work on.
